### DISCUSSION TITLE
AWS CLI throws error while pulling a json file

### DISCUSSION THREAD

**Discussion by ravisinha0506** [aws/aws-cli#6861] [Q&A] [answered]
### Describe the bug

AWS CLI ( `aws-cli/2.4.15 Python/3.8.8 Darwin/19.6.0 exe/x86_64 prompt/off` ) fails while executing below command:

```
aws ssm create-document --name "EKSLogCollector" --document-type "Command" --content https://raw.githubusercontent.com/awslabs/amazon-eks-ami/master/log-collector-script/linux/eks-ssm-content.json

```

and fails with error:

```
An error occurred (InvalidDocumentContent) when calling the CreateDocument operation: JSON not well-formed. at Line: 1, Column: 6
```

However, the same command works fine when the file is provided locally like this:

```
aws ssm create-document --name "EKSLogCollector" --document-type "Command" --content file:///Users/$user/Documents/eks-ssm-content.json
````

### Expected Behavior

AWS CLI shouldn't fail while pulling the JSON file.

### Current Behavior

Currently it fails with below error:

```
 JSON not well-formed. at Line: 1, Column: 6
```

### Reproduction Steps

Install CLI version `aws-cli/2.4.15 Python/3.8.8 Darwin/19.6.0 exe/x86_64 prompt/off` and execute below commands:

```
aws ssm create-document --name "EKSLogCollector" --document-type "Command" --content file:///Users/$user/Documents/eks-ssm-content.json
```

### CLI version used

aws-cli/2.4.15 Python/3.8.8 Darwin/19.6.0 exe/x86_64 prompt/off

### Environment details (OS name and version, etc.)

Mac OS Catalina ( 10.15.7 )

**Comment by kdaily** [comment, marked as answer]
Hi @ravisinha0506,

Thanks for your post. This is a change made to the AWS CLI v2 client. You can read more about this change here:

https://docs.aws.amazon.com/cli/latest/userguide/cliv2-migration.html#cliv2-migration-paramfile

Since this is a guidance question, I'm going to move this to the GitHub discussions section. If this answers your question, marking this as the answer would be appreciated!
